packages/era/src/era.ts

```diff
@@ -82,11 +70,13 @@ export const readBeaconState = async (eraData: Uint8Array) => {
     throw EthereumJSErrorWithoutCode(`expected CompressedBeaconState type, got ${stateEntry.type}`)
   }
   const stateSlot = indices.stateSlotIndex.startSlot
-  if (stateSlot < ForkSlots.Altair) return Phase0BeaconState.decode(data.data as Uint8Array)
-  else if (stateSlot < ForkSlots.Bellatrix) return AltairBeaconState.decode(data.data as Uint8Array)
-  else if (stateSlot < ForkSlots.Capella)
-    return BellatrixBeaconState.decode(data.data as Uint8Array)
-  else if (stateSlot < ForkSlots.Deneb) return CapellaBeaconState.decode(data.data as Uint8Array)
+  if (stateSlot < ssz.ForkSlots.Altair) return ssz.Phase0BeaconState.decode(data.data as Uint8Array)
```

Comment:
I guess this if else still works if two/more forks are scheduled on the same slot, but adding a comment to that effect will be helpful to a third reader or may be (much) later to ourselves as well lol

Reply:
I guess I'm not sure what you mean.  This code can't stay static and will have to update with more forks down the line.  There's probably a better way to do this but I haven't set up a `forkFinder` sort of helper method and I really want to do another pr to `meth` to put all the fork specific containers under a `ssz.ETH2_TYPES['fork'].containerType` structure and not the way it is now where everything is exported from the base